ttnn/cpp/ttnn/operations/conv/conv2d/conv2d.hpp

```diff
@@ -53,54 +53,6 @@ struct Conv2dConfig {
     bool enable_weights_double_buffer = false; // Used on for block sharded convolutions
     bool enable_split_reader = false;
     bool enable_subblock_padding = false;
-    static constexpr auto attribute_names = std::make_tuple(
```

Comment:
@mywoodstock [said](https://github.com/tenstorrent/tt-metal/pull/14210#discussion_r1815797399) it's no long needed.

Reply:
I'm using it, if you run tracy profile this will give you info on which setup conv was run with.
@mywoodstock is there maybe a better workflow that I don't know about?

Reply:
Are these attributes necessary for tracy to run correctly?

Reply:
No it will run w/o them, but when you look at the trace results you won't have info on these params that were used to run the conv, and I find that very helpful.
You have same thing on MMs

Reply:
Can we pls get this back?